pkg/pool-stable/test/foundry/RoundingDirectionEdgeCases.t.sol

```diff
@@ -79,6 +83,18 @@ contract RoundingDirectionStablePoolEdgeCasesTest is BasePoolTest {
                 )
             )
         );
+
+        // poolArgs is used to check pool deployment address with create2.
+        poolArgs = abi.encode(
```

Comment:
Hm, this looks like an implicit return argument to me.

If it doesn't cause too much trouble I would consider returning this explicitly and storing it in the base pool test. Otherwise we're kinda breaking the encapsulation of the tests. It's not terrible at this point, but I don't think it's ideal.

Reply:
Done!